cg/cli/upload/mutacc.py

```diff
@@ -22,7 +22,7 @@ def process_solved(context, case_id, days_ago, customers, dry_run):
     click.echo(click.style("----------------- PROCESS-SOLVED ----------------"))
```

Comment:
I want to replace all of those with log messages also but that's for another PR 🍇

Reply:
Be the change you want to see in the world :rainbow: